views/scss/qtiCreator/_item_buttons.scss

```diff
@@ -0,0 +1,37 @@
+// Vertical item (default)
+.add-options {
+    margin: 15px 5px 8px 0;
+}
+.add-option {
+    display: inline-block;
+    vertical-align: top;
+    position: relative;
+    margin-left: 2px;
+    padding: 5px 16px 5px 6px;
+    overflow: hidden;
+
+    @include rtl {
+        padding-right: 6px;
+        padding-left: 16px;
+    }
+
+    .icon-add {
+        font-size: 1.8rem;
+        left: 6px;
+        position: absolute;
+        top: 5px;
```

Comment:
This fix breaks the widget Block - inlineChoice
![inlineChoice](https://user-images.githubusercontent.com/11692751/134148729-4f8e7e4c-6509-4754-a0b4-f0486e8aa4a6.png)

Reply:
good catch, checking